test/markup/css/pseudo-selector.txt

```diff
@@ -22,3 +22,47 @@ p::first-letter {
   font-weight: bold;
   color: brown;
 }
+.dark\:hover\:bg-sky-500:hover {
```

Comment:
Two or three (tops) of these longer examples should be sufficient unless there is some big difference here I'm missing.

Reply:
okay ill reduce the newer tc